packages/create-svelte/shared/+eslint/.eslintignore

```diff
@@ -1,13 +1,12 @@
-.DS_Store
+/package
```

Comment:
wouldn't this be considered "Output"?

I think if you changed the "Lockfiles" head to "Package managers" then you could put `node_modules` there too

Reply:
I wasn't certain about the `/packages` directory. When is it used? The package template uses the `dist` directory.

Reply:
perhaps we don't need it anymore. it may be leftover. I think svelte-kit package used to output to that location